collections/README.md

```diff
@@ -0,0 +1,106 @@
+# STAC API - Collections
+
+- [STAC API - Collections](#stac-api---collections)
+	- [Link Relations](#link-relations)
+	- [Endpoints](#endpoints)
+	- [Example](#example)
+	- [Conformance](#conformance)
+
+- **OpenAPI specification:** Missing
+- **Conformance URI:** <http://stacspec.org/spec/api/1.0.0-beta.1/extensions/collections>
+- **Extension [Maturity Classification](../extensions.md#extension-maturity):** Pilot
+- **Dependencies**: [STAC API - Core](../core)
+
+A STAC API can return information about all STAC [Collections](../stac-spec/collection-spec/collection-spec.md) available using a link
+from the landing page that uses the `data` rel, which links to an endpoint called `/collections`. Individual STAC collections can be accessed
+by providing the Collection `id` as a path past that endpoint: `/collections/{collectionId}`.
+
+**NOTE**: *This conformance class is directly based on the [Features Collection](http://docs.opengeospatial.org/is/17-069r3/17-069r3.html#_collections_)
+section of OAFeat, which is in the process of becoming a 'building block in [OGC API - Common - Part 2: Geospatial 
+Data](http://docs.opengeospatial.org/DRAFTS/20-024.html) as the [Collections requirements 
+class](http://docs.opengeospatial.org/DRAFTS/20-024.html#rc_collections-section). Once the Common version is released we will 
+aim to align with it. But it still seems to be in flux.*
+
+## Link Relations
+
+The following Link relations should exist in the Landing Page (root).
```

Comment:
Was this intended to have Link relations for the Landing page here on the Collections endpoint?

Reply:
yes. We probably also need to clarify which link relations should exist on the /collections and /collections/{collectionid} endpoints for both this and ogcapi-features.